awesomemotive/file-upload-types
# Gulp build task: exclude composer dev-packages

use gulp-composer to run composer

fixes #65

test steps:
- clone this repository (or if you already have cloned, remove `node_modules`, `build` and `vendor` directories)
- run `npm install`
- run `gulp build`
- unzip file from `build directory`
- see if the plugin works
- see if `vendor` directory from zip has only autoloader dependencies
